base/set.jl

```diff
@@ -282,29 +284,42 @@ julia> issubset([1, 2, 3], [1, 2])
 false
 ```
 """
-function issubset(l, r)
+function ⊆(l, r)
     for elt in l
         if !in(elt, r)
             return false
         end
     end
     return true
 end
-
 # use the implementation below when it becoms as efficient
 # issubset(l, r) = all(_in(r), l)
 
-const ⊆ = issubset
-⊊(l::Set, r::Set) = <(l, r)
-⊈(l::Set, r::Set) = !⊆(l, r)
-⊇(l, r) = issubset(r, l)
-⊉(l::Set, r::Set) = !⊇(l, r)
-⊋(l::Set, r::Set) = <(r, l)
+const issubset = ⊆
```

Comment:
For all other unicode aliases, the ASCII name is the core function name, so this should be reversed.

Reply:
OK I understand, I just did it so that the printing of `⊆` and `⊇` gets more consistent.